Simple Calculator/Calculator.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+
+namespace Simple_Calculator
+{
+    public partial class Calculator : Form
+    {
+        //variables which are being used
+        string valOne = "";
+        string valTwo = "";
+        string result = "";
+        string ArithOperator;
+        
+        public Calculator()
+        {
+            InitializeComponent();
+        }
+
+        private void button_Click(object sender, EventArgs e)
+        {
+            Button button = (Button)sender;
+            txtResult.Text = txtResult.Text + button.Text;
+        }
+
+        private void operators_Click(object sender, EventArgs e)
+        {
```

Comment:
get the return value of `CalCulateResult` and update the `txtResult` field in this method.

Reply:
> get the return value of `CalCulateResult` and update the `txtResult` field in this method.

i couldnt understand how to do it.